### ISSUE TITLE
Docs: update the token failure example to OctoKit 404 failure

### ISSUE THREAD

**Issue by olleolleolle** [skywinder/github-changelog-generator#435]
This part of the readme refers to a failure which won't happen with OctoKit: 

https://github.com/skywinder/github-changelog-generator/#github-token

**Comment by deohtee** [comment]
Is this the 404 error that is supposed to show with OctoKit?

```
API rate limit exceeded for deohtee.
See: https://developer.github.com/v3/#rate-limiting

```

Want to double check before submitting a PR.

**Comment by olleolleolle** [comment]
@deohtee Yes, this is the kind of error that we would like to talk about in this part of the README. There may be other outputs, but this communicative error message (including that link about Rate Limiting, suuuuper) guides the user to the right solution quicker.

To be clear, this section is now inaccurate:

```
So, if you got an error like this:

! /Library/Ruby/Gems/2.0.0/gems/github_api-0.12.2/lib/github_api/response/raise_error.rb:14:in `on_complete'

```

**Comment by skywinder** [comment]
Related to #482
